frontend/src/services/member/Onboarding.js

```diff
@@ -0,0 +1,32 @@
+// const BASE_URL = 'https://some-backend-api-url.com'
+
+// eslint-disable-next-line no-unused-vars
+export const postSubmitApplication = async (data) => {
+  // const response = await fetch(`${BASE_URL}/api/application`,data)
+  // const data = await response.json()
+  const exampleResponse = {
+    status: 204,
+    headers: {
+      'Content-Type': 'application/json',
+      'X-Custom-Header': 'Custom Value',
```

Comment:
Why the custom headers?

Reply:
I'll remove it I just used some random sample response.